### PR TITLE
feat: add copy images method in pkg.

### PR DESCRIPTION
**Pull request by Jeremy-boo**
# Changes

add copy image method to pkg which from github.com/containers/image

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] [`spec` PR link](https://github.com/katanomi/spec) included
- [x] Follows the [commit message standard](https://github.com/katanomi/spec/blob/main/CONTRIBUTING.md#commits)
- [x] Meets the [contributing guidelines](https://github.com/katanomi/spec/blob/main/CONTRIBUTING.md) (including
  functionality, content, code)
- [ ] Test cases with documentation and functionality works as expected using current and related github repos (MUST deploy and check)
- [ ] Release notes block below has been filled in or deleted (only if no user facing changes)

# Release Notes

### CODE REVIEW

File: .golangci.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 run:
   timeout: 10m
 
-  build-tags: []
+  build-tags: [containers_image_openpgp]
```
Comment (Jeremy-boo) [inline .golangci.yaml:4]: github.com/containers/image 编译需要cgo 环境，添加 build-tags 参数，使其不强依赖 cgo 环境

File: base.mk [hunk 2]
```diff
@@ -39,16 +39,16 @@ fmt: ##@Development Run go fmt against code.
 	go fmt ./...
 
 vet: ##@Development Run go vet against code.
-	go vet ./...
-	go vet -tags e2e ./...
+	go vet -tags containers_image_openpgp ./...
```
Comment (Jeremy-boo) [inline base.mk:42]: 同上

File: .golangci.yaml (hunk 1 above)
Comment (l-qing) [reply .golangci.yaml:4]: 跟其他地方的数组参数形式保持一致？
```
map:
  - key1
  - key2
```

File: .ko.yaml [hunk 3]
```diff
@@ -1,2 +1,3 @@
 # Use :nonroot base image for all containers
 defaultBaseImage: katanomi/distroless-static:nonroot
+
```
Comment (l-qing) [inline .ko.yaml:3]: 这个有必要？

File: .golangci.yaml (hunk 1 above)
Comment (l-qing) [reply .golangci.yaml:4]: 有相关的文档链接么？

File: base.mk (hunk 2 above)
Comment (l-qing) [reply base.mk:42]: 1. 同一个内容需要改 4 处，是不是可以换成一个变量，这样下次改起来就轻松了？
2. base.mk 不应该在代码仓库中直接改，而是由 https://github.com/katanomi/hack/blob/main/base.mk 统一修改？
3. 新增这个 tag 后，gitlab 上的代码是否也需要加？

File: .golangci.yaml (hunk 1 above)
Comment (Jeremy-boo) [reply .golangci.yaml:4]: 嗯啦，我贴一下

**Pull request closed without merging**